src/index.html

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Colored blocks</h1>
+  <div id="square1"></div>
```

Comment:
use classes for styling, id is used for special purposes only (js, anchors and so on)